vowpalwabbit/slim/include/vw_slim_predict.h

```diff
@@ -297,10 +297,10 @@ class vw_predict
 
     // VW performs the following transformation as a side-effect of looking for duplicates.
     // This affects how interaction hashes are generated.
-    std::vector<std::string> vec_sorted;
-    for (const std::string& interaction : _interactions)
+    std::vector<std::vector<uint8_t>> vec_sorted;
+    for (const std::vector<uint8_t>& interaction : _interactions)
```

Comment:
You could make the loop arg a non-ref, so you get a by value copy then sort that

Reply:
Right, this would avoid sorting the original interaction vectors. Updated that